packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -409,8 +408,8 @@ export class ProfilesUtils {
                     if (await vscode.env.openExternal(credentialManagerInstallURL)) {
                         const refreshMessage = localize(
                             "ProfilesUtils.promptAndHandleMissingCredentialManager.refreshMessage",
-                            `After installing the extension, please make sure to reload
-                             your VS Code window in order to start using the installed credential manager`
+                            `After installing the extension, please make sure to reload your VS Code window in order
+                             to start using the installed credential manager`
```

Comment:
Not sure why the extra spacing doesn't show, but the notification looks good
![image](https://github.com/zowe/vscode-extension-for-zowe/assets/37381190/a9c3f6bc-d5f1-4d7f-8b59-0808932af1b6)


Also, should the button say `Reload` instead of `Refresh`

Reply:
hmm that makes sense! I think Refresh suits it better :)